starter-code/src/app/component/my-movie/my-movie.component.html

```diff
@@ -0,0 +1,29 @@
+<div>
+  <p>
+    my-movie works!
+  </p>
+  <div>
+    <p>{{movieName.title}}</p>
```

Comment:
Now we know that at the beginning `movieName` it will be undefined. Don't forget to handle this error with a `*ngIf`